### DISCUSSION TITLE
Briefly explain about Circular Queue.

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#46] [Q&A] [answered]
brief

**Comment by fcasamento** [comment, marked as answer]
A Circular Queue is a special version of queue where the last element of the queue is connected to the first element of the queue forming a circle.

![circular-queue-implementation](https://user-images.githubusercontent.com/19937845/205522810-a4e2dfe4-64f5-434c-8e7c-60d40b91c197.gif)

**Reply by fcasamento** [reply]
@ankit-akash Mark this as an answer as well, we can help each other. 🙏🏻 Pls
